src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
```

Comment:
Avoid beginning method implementation with an empty line. Please remove the redundant empty line at the start of the `handle` method.